exercises/minesweeper/MinesweeperTest.cs

```diff
@@ -1,142 +1,240 @@
-﻿using Xunit;
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
 
 public class MinesweeperTest
 {
     [Fact]
-    public void Zero_sized_board()
+    public void No_rows()
     {
-        var input = "";
-        var expected = "";
+        var input = new [] 
```

Comment:
I think no rows should mean an empty array, not an array with an empty string. Could you replace that (also for the expected value)?

Reply:
Sure!